apps/dashboard/src/components/primitives/sonner.tsx

```diff
@@ -81,6 +82,37 @@ const ToastClose = ({ className, ...props }: React.HTMLAttributes<HTMLButtonElem
   );
 };
 
+const PromoteToast = ({
```

Comment:
please do not create the specific promote toast, all this code should be in the place where you want to show it, meaning in the `const onPromoteSuccess = () => {` functions